src/applications/ezr/config/chapters/insuranceInformation/policyInformation.js

```diff
@@ -30,10 +30,16 @@ export default {
     insuranceName: {
       'ui:title': content['insurance-provider-name-label'],
       'ui:webComponentField': VaTextInputField,
+      'ui:errorMessages': {
+        pattern: 'Enter the insurance provider name',
+      },
     },
     insurancePolicyHolderName: {
       'ui:title': content['insurance-policyholder-name-label'],
       'ui:webComponentField': VaTextInputField,
+      'ui:errorMessages': {
+        pattern: 'Enter the policyholder\u2019s name',
```

Comment:
🎨 Is the `\u2019` escape code the only way to make an apostrophe here? IMHO this could be clearer as
`        pattern: "Enter the policyholder's name",`
or something?

Reply:
Welp, if that's the consistent approach, that's fair!